pibooth_sound_effects.py

```diff
@@ -47,65 +47,66 @@ def pibooth_cleanup():
 
 #--- Wait State ---------------------------------------------------------------
 @pibooth.hookimpl
-def state_wait_enter(app):
+def state_wait_enter(app, cfg):
```

Comment:
Remove cfg from all hooks signature when it is not necessary

Reply:
fix